app/views/results/_defendant_appeals.html.haml

```diff
@@ -0,0 +1,27 @@
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('search.result.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.name')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.date_of_birth')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.prosecution_case_reference')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.arrest_summons_number')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.court_application_title')
+  %tbody.govuk-table__body
+    - results.each do |defendant|
+      %tr.govuk-table__row
+        %td.govuk-table__cell
+          = link_to defendant.name, defendant_link_path(defendant, defendant.prosecution_case_reference), class: 'govuk-link'
+        %td.govuk-table__cell
+          = l(defendant.date_of_birth&.to_date)
+        %td.govuk-table__cell
+          = link_to defendant.prosecution_case_reference, prosecution_case_path(defendant.prosecution_case_reference), class: 'govuk-link'
+
+        %td.govuk-table__cell
+          = defendant.arrest_summons_number
+        %td.govuk-table__cell
+          - if defendant&.application_summaries&.present?
```

Comment:
It would be good to have a little test coverage here. Maybe a feature spec that uses VCR to pull in a result with a couple of court applications, and assert that when the feature flag is switched on their titles are visible?